### ISSUE TITLE
Side effect on fix for issue #13371 for Digimon Adventure PSP NPJH-50686 Cutscene videos

### ISSUE THREAD

**Issue by LPatamon** [hrydgard/ppsspp#13379]
### What happens?
First off thank you very much for fixing the crashing when cutscenes videos on Digimon Adventure PSP NPJH-50686 Cutscene videos played.

But there was a tiny side effect:

![image](https://user-images.githubusercontent.com/6118529/92224881-12582f00-ee68-11ea-9cc4-fb16bfd27f9d.png)

The line on bottom the videos is back, but this time is kinda curious since it only happens when you play a video after loading your progress, but after that if you play that video or any other you had unlocked, the line doesn't appears o.o

This didn't happened before, before was the line present all time till it got fixed for issue #12186 that I reported as well. It was fixed for v1.8.0-623-gc7798715a (no download links, they appeared on the next build the same date) with the comment **"Fix issue preventing video to work correctly in Digimon. Fixes #12186 (due to BlockTransferAllowCreateFB not updating the framebuffer range speedhack)."**

And yes. it has to do with the framebuffer.

### What *should* happen?

There shouldn't be no line on the bottom of the videos

### What hardware, operating system, and PPSSPP version?  On desktop, GPU matters for graphical issues.

3Ghz Core i5-3330 (Quad Core), 4GB Ram and 2GB GDDR5 Gigabyte Nvidia GT1030 on Windows 7 Ultimate 32 bits Desktop PC.

PPSSPP Version: v1.10.3-571-g619009be9

### Notes

Guess it must be just a tiny fix, nothing major.

**Comment by hrydgard** [comment]
That is very weird, thanks for reporting.

**Comment by LPatamon** [comment]
> 
> 
> That is very weird, thanks for reporting.

Indeed is kinda odd. Also I noticed that the line on the videos is always present when you play and a cutscene video is played (for example a digivolution sequence video), but if you go to the video library to watch the cutscenes you unlocked, the line only appears on the first video you load. After that any other video you load, will appear without the line but only on the video library.

Guess our old pal the Framebuffer is a little troll huh?

**Comment by LPatamon** [comment]
Yes. but the lines were fixed on x2 resolution and above on v1.8.0-623-gc7798715a along with another issue the game had back then with the videos which was that when I finished playing a video and then played another, the screen on the emulator went black but the video sound could be heard.

When you fixed that issue, the line on resolution x2 was fixed as well. Also take a look at this video. 

https://www.youtube.com/watch?v=pWP_zCdJoTU

As you can see when playing videos through the library, only the first time I play a video the line appears on x2 resolution, if I play the video (or any other video from the library) again, the line doesn't appears. This only happens on the library. On the normal gameplay, the line is always there on the videos, thing that was fixed on July, 2019 on v1.8.0-623-gc7798715a. Since then there was no line on the videos on any window resolution, not matter if you played the videos through the library or that they played during your progress through the game.

**Comment by IrfanH495** [comment]
Sorry I want to ask, does this game feel heavy in some parts.?
this is my first time playing this game, experienced a decrease in fps in some parts Vulkan & opengl.
![Screenshot_20240112-020309_PPSSPP](https://github.com/hrydgard/ppsspp/assets/78854859/da11129c-d9ec-4a9b-981a-8fc8356f6b22)
![Screenshot_20240112-022538_PPSSPP](https://github.com/hrydgard/ppsspp/assets/78854859/1efaf10e-1cae-4218-9c40-5ca5507484f0)

**Comment by hrydgard** [comment]
The cartoon effect the game does is not super cheap - to confirm that it's the issue, you can try reducing rendering resolution one step and see if that improves performance.

**Comment by IrfanH495** [comment]
all default settings, 1xpsp no change fps still below 25
i tried playstore version and build, 
device samsung m20
![Screenshot_20240112-034545_PPSSPP](https://github.com/hrydgard/ppsspp/assets/78854859/0cf86dec-f651-4c07-9fbe-14e5a7311500)
Yes, it's true that the cartoon image is very good for me,
maybe my device is not very powerful.  I will try another device.

**Comment by Deka710** [comment]
> all default settings, 1xpsp no change fps still below 25 i tried playstore version and build, device samsung m20 ![Screenshot_20240112-034545_PPSSPP](https://private-user-images.githubusercontent.com/78854859/296047879-0cf86dec-f651-4c07-9fbe-14e5a7311500.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzE3NDU2MzgsIm5iZiI6MTczMTc0NTMzOCwicGF0aCI6Ii83ODg1NDg1OS8yOTYwNDc4NzktMGNmODZkZWMtZjY1MS00YzA3LTlmYmUtMTRlNWE3MzExNTAwLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDExMTYlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMTE2VDA4MjIxOFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTlkZWQ0MjZiMTU5YThiNjEzNDhkMWQzZTJhODhkYTVmNzU3MDFjZjgxYzNiOWEzYTFkNTk5ZjgzYmFmN2Y3OTkmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.SWyBOG6oI8elnkjB8AERi9nQIKpViu94dDUGH0uPZTY) Yes, it's true that the cartoon image is very good for me, maybe my device is not very powerful. I will try another device.

overclock the cpu in system setting helps inbmy case
